src/main/kotlin/org/rust/ide/spelling/RsNameIdentifierOwnerTokenizer.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.spelling
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.util.PsiTreeUtil
+import com.intellij.spellchecker.inspections.IdentifierSplitter
+import com.intellij.spellchecker.tokenizer.TokenConsumer
+import com.intellij.spellchecker.tokenizer.Tokenizer
+import org.rust.lang.core.psi.RS_RAW_PREFIX
+import org.rust.lang.core.psi.ext.RsNameIdentifierOwner
+import org.rust.lang.core.psi.unescapeIdentifier
+
+// Inspired by `PsiIdentifierOwnerTokenizer`
+object RsNameIdentifierOwnerTokenizer : Tokenizer<RsNameIdentifierOwner>() {
+
+    override fun tokenize(element: RsNameIdentifierOwner, consumer: TokenConsumer) {
+        val identifier = element.nameIdentifier ?: return
+        val range = identifier.textRange
+        if (range.isEmpty) return
+
+        var offset = range.startOffset - element.textRange.startOffset
```

Comment:
@ortem maybe it's better to be closer to the original code (I mean the code of `PsiIdentifierOwnerTokenizer`)? It can make a future comparison of the original code and our easier for example in case of changes in original code

Reply:
OK, it makes sense.